osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs

```diff
@@ -135,7 +135,9 @@ public SensitivitySetting()
 
         private class SensitivitySlider : OsuSliderBar<double>
         {
-            public override LocalisableString TooltipText => Current.Disabled ? MouseSettingsStrings.EnableHighPrecisionForSensitivityAdjust : $"{base.TooltipText}x";
+            protected override LocalisableString GetTooltipText(double value) => Current.Disabled
+                ? MouseSettingsStrings.EnableHighPrecisionForSensitivityAdjust
+                : $"{base.GetTooltipText(value)}x";
```

Comment:
After thoroughly checking each usage, this actually will no longer work properly as `GetTooltipText` updates on value change only, not on disable change as well.

Unsure if `GetTooltipText` should be updated on disable callback, or revert the pattern back to `TooltipText` to handle more scenarios than that.

Reply:
Well at this point I'd personally revert [abba49f](https://github.com/ppy/osu/pull/17073/commits/abba49fd8f8bbe15b93ace48809bc342b486488a) and use `TooltipText` normally because yes, you could subscribe to `DisabledChanged`, but then you'd have to add the disable value to `GetTooltipText()` as a param to not be weird. Dunno, that's my feeling anyway.

Reply:
Revert all usages, not just the broken one? Does that mean we'll leave the two different methods of usage for now?

Reply:
Revert all usages and let `OffsetSliderBar` override `TooltipText` instead, should work just fine I believe.

Reply:
Yeah I'd say revert all and remove `GetTooltipText()` entirely. As it turns out framework-side [`TooltipContainer` makes `GetTooltipText()` unnecessary](https://github.com/ppy/osu-framework/blob/efcbe56398543652089fbebc96d1fda30f9e3353/osu.Framework/Graphics/Cursor/TooltipContainer.cs#L167-L171) as content is going to get potentially refreshed every frame.

Reply:
Have applied changes, seems to work.